src/shared/styles/styledComponents/SmallNumberInput.ts

```diff
@@ -30,10 +34,27 @@ export const StyledSmallNumberInput = styled(Box)`
         border-color: ${variables.palette.primary};
       }
     }
+
+    .MuiInputBase-input {
+      padding: ${theme.spacing(0.7, 0)};
+      text-align: center;
+      max-width: 7rem;
+      overflow: scroll;
```

Comment:
I think `overflow` can be safely removed

Reply:
Do we have a max amount of characters this input would handle? Since for both current ones, there's no max value defined, I went with the `scroll` prop here

Reply:
No, no set amount of characters. We can keep this based on your comment